src/main/java/com/exasol/repository/ReleaseChangesLetter.java

```diff
@@ -0,0 +1,164 @@
+package com.exasol.repository;
+
+import static com.exasol.ReleaseRobotConstants.*;
+
+import java.time.LocalDate;
+import java.util.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * This class represents a changes file's content.
+ */
+public class ReleaseChangesLetter {
+    private final String fileName;
+    private String header = null;
+    private String body = null;
+    private String versionNumber = null;
+    private LocalDate releaseDate = null;
+    private final List<Integer> ticketNumbers = new ArrayList<>();
+
+    /**
+     * Create a new {@link ReleaseChangesLetter} from a string.
+     * 
+     * @param fileName name of the changes file
+     * @param content file's content as a string
+     */
+    public ReleaseChangesLetter(final String fileName, final String content) {
+        this.fileName = fileName;
+        if (content != null && content.length() > 1) {
+            parseContent(content);
+        }
+    }
+
+    /**
+     * Get a name of the changes file.
+     * 
+     * @return name of the changes file
+     */
+    public String getFileName() {
+        return this.fileName;
+    }
+
+    /**
+     * Get a version number.
+     * 
+     * @return version number as a string
+     */
+    public Optional<String> getVersionNumber() {
+        return this.versionNumber == null ? Optional.empty() : Optional.of(this.versionNumber);
+    }
+
+    /**
+     * Get a release date.
+     * 
+     * @return release date
+     */
+    public Optional<LocalDate> getReleaseDate() {
+        return this.releaseDate == null ? Optional.empty() : Optional.of(this.releaseDate);
+    }
+
+    /**
+     * Get a release letter header.
+     * 
+     * @return header as a string
+     */
+    public Optional<String> getHeader() {
+        return this.header == null ? Optional.empty() : Optional.of(this.header);
+    }
+
+    /**
+     * Get a release letter body.
+     * 
+     * @return body as a atring
+     */
+    public Optional<String> getBody() {
+        return this.body == null ? Optional.empty() : Optional.of(this.body);
```

Comment:
```suggestion
        return Optional.ofNullable(this.body);
```

Reply:
You don't need to do it, I check the same issues over the code anyway :D

Reply:
Force of habbit.